### PR TITLE
SISRP-26827 - 7.2 - Advising Resources Card for My Dashboard (Advisor) - Reorganize card

### PR DESCRIPTION
**Pull request by dfkaye**
https://jira.berkeley.edu/browse/SISRP-26827

### CODE REVIEW

File: src/assets/templates/widgets/advisor/advising_links.html [hunk 1]
```diff
@@ -1,63 +1,94 @@
...
+    </li>
+    <li data-ng-if="csLinks.ucClassSearch.url">
+      <div
+        data-cc-campus-solutions-link-item-directive
+        data-link="csLinks.ucClassSearch"
+      ></div>
+    </li>
+  </ul>
+
+  <h3>Student Specific Links</h3>
```
Comment (johncrossman) [inline src/assets/templates/widgets/advisor/advising_links.html]: @dfkaye student-specific links will never surface on dashboard, right?

Comment (johncrossman) [inline src/assets/templates/widgets/advisor/advising_links.html]: @dfkaye  ah, these links don't rely on a `student_id` in context. If so, ignore comment above.

**Comment by johncrossman** [comment]
Looks good. Thanks, @dfkaye

**Comment by dfkaye** [comment]
Thanks @johncrossman :)

**Pull request merged**